weblab/repocache/tests/test_populate.py

```diff
@@ -1,28 +1,37 @@
 import pytest
 
+from accounts.models import User
 from core import recipes
 from repocache.populate import populate_entity_cache
 
 
 @pytest.mark.django_db
 class TestPopulate:
-    def test_populate(self, model_with_version):
+    def test_populate(self, model_with_version, helpers):
         model_with_version.repo.tag('v1')
-
         populate_entity_cache(model_with_version)
-
         cached = model_with_version.cachedentity
-
         assert cached is not None
-        latest = model_with_version.repo.latest_commit
 
-        version = cached.versions.get()
-        assert version.sha == latest.sha
-        assert version.message == latest.message
-        assert version.timestamp == latest.timestamp
+        latest = model_with_version.repo.latest_commit
+        version1 = cached.versions.get()
 
+        assert version1.sha == latest.sha
+        assert version1.message == latest.message
+        assert version1.timestamp == latest.timestamp
+        assert version1.master_filename == latest.master_filename is None
```

Comment:
I'm not sure this will work as expected - you might need to split into 2 asserts? I'm not sure what it's actually doing!